training_config/transformer_squad.bert-base.jsonnet

```diff
@@ -0,0 +1,60 @@
+local transformer_model = "bert-base-cased";
+local epochs = 3;
+local batch_size = 8;
+
+// This defines the number of instances, not the number of questions. One question can end up as
+// multiple instances.
+local number_of_train_instances = 87825;
```

Comment:
Could you leave a note on how to compute this?

Reply:
I think I'd rather wait for @matt-gardner's change to `fromParams`, so that this is not necessary anymore.